operators/go/routeagent_controller.go.nolint

```diff
@@ -145,8 +145,7 @@ func newPodForCR(cr *submarinerv1alpha1.Routeagent) *corev1.Pod {
 			Containers: []corev1.Container{
 				{
 					Name: "submariner-routeagent",
-					// TODO: Use var here
-					Image: "submariner-route-agent:local",
+					Image: cr.Spec.Image,
```

Comment:
routeAgent spec does not have Spec.Image field, right?

No need to add, I will collapse to submariner CRD.

Reply:
I’ve added it anyway, just so I can test from Quay ;-).